### PR TITLE
Add JS tools related to the documentation

### PR DESCRIPTION
**Pull request by JuliaEvseeva**
This PR brings tools for the interactive diagram and code language switcher.

They were placed in the `tools` folder and will be automatically added to the main `spine.io` repository when the submodule will be updated. The commit history of the files was preserved.

The needed styles have been also added to this repository and should be refactored in the main repository to avoid duplication.

### CODE REVIEW

File: _libs/js.cookie.min.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
...
...mber"==typeof(i=e({},o,i)).expires&&(i.expires=new Date(Date.now()+864e5*i.expires)),i.expires&&(i.expires=i.expires.toUTCString()),t=encodeURIComponent(t).replace(/%(2[346B]|5E|60|7C)/g,decodeURIComponent).replace(/[()]/g,escape);var c="";for(var u in i)i[u]&&(c+="; "+u,!0!==i[u]&&(c+="="+i[u].split(";")[0]));return document.cookie=t+"="+n.write(r,t)+c}}return Object.create({set:r,get:function(e){if("undefined"!=typeof document&&(!arguments.length||e)){for(var t=document.cookie?document.cookie.split("; "):[],o={},r=0;r<t.length;r++){var i=t[r].split("="),c=i.slice(1).join("=");try{var u=decodeURIComponent(i[0]);if(o[u]=n.read(c,u),e===u)break}catch(e){}}return e?o[e]:o}},remove:function(t,n){r(t,"",e({},n,{expires:-1}))},withAttributes:function(n){return t(this.converter,e({},this.attributes,n))},withConverter:function(n){return t(e({},this.converter,n),this.attributes)}},{attributes:{value:Object.freeze(o)},converter:{value:Object.freeze(n)}})}({read:function(e){return'"'===e[0]&&(e=e.slice(1,-1)),e.replace(/(%[\dA-F]{2})+/gi,decodeURIComponent)},write:function(e){return encodeURIComponent(e).replace(/%(2[346BF]|3[AC-F]|40|5[BDE]|60|7[BCD])/g,decodeURIComponent)}},{path:"/"})}));
```
[the remark this answers is not in the document]
Comment (JuliaEvseeva) [reply _libs/js.cookie.min.js:2]: @sonatype-lift ignore

**Comment by JuliaEvseeva** [comment]
@armiol PTAL.

File: _diagrams/README.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+To have several classes for the one element add `.g-caption.command-dispatcher`. 
+The result will be `class="g-caption command-dispatcher"`.
+
+A layer name without a prefixed period will be used as ID.
+For example, `g-caption.command-dispatcher` will result `id="g-caption" class="command-dispatcher"`.
+
+## Export SVG
+
+1. Select the diagram artboard.
+2. Go to the right sidebar;
```
Comment (armiol) [inline _diagrams/README.md]: Let's make it consistent and use `.` at the end of the points. It's here and also below.

**Review by armiol — APPROVED** [review]
@JuliaEvseeva LGTM with a small comment to address before merging.

**Pull request merged**